I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 554, in script call
    call gameloop
  File "game/gameloop.rpy", line 9, in script call
    call maploop
  File "game/script.rpy", line 580, in script call
    call herbs
  File "game/script.rpy", line 684, in script call
    call maploop
  File "game/script.rpy", line 582, in script call
    call river
  File "game/script.rpy", line 704, in script
    if game.rollevent == 1: # 1 of the 4 actions
  File "game/script.rpy", line 704, in <module>
    if game.rollevent == 1: # 1 of the 4 actions
AttributeError: 'Game' object has no attribute 'rollevent'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 554, in script call
    call gameloop
  File "game/gameloop.rpy", line 9, in script call
    call maploop
  File "game/script.rpy", line 580, in script call
    call herbs
  File "game/script.rpy", line 684, in script call
    call maploop
  File "game/script.rpy", line 582, in script call
    call river
  File "game/script.rpy", line 704, in script
    if game.rollevent == 1: # 1 of the 4 actions
  File "/Applications/renpy-7.3.5-sdk/renpy/ast.py", line 1832, in execute
    if renpy.python.py_eval(condition):
  File "/Applications/renpy-7.3.5-sdk/renpy/python.py", line 2059, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Applications/renpy-7.3.5-sdk/renpy/python.py", line 2052, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 704, in <module>
    if game.rollevent == 1: # 1 of the 4 actions
AttributeError: 'Game' object has no attribute 'rollevent'

Darwin-19.6.0-x86_64-i386-64bit
Ren'Py 7.3.5.606
sunsunsun 1.0
Wed Jul  7 16:25:54 2021
